### PR TITLE
use tagname without artifact prefix for releases

### PR DESCRIPTION
**Pull request by lanwen**
This change is [Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/124)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -282,6 +282,19 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <artifactId>maven-release-plugin</artifactId>
+                <configuration>
+                    <tagNameFormat>v@{project.version}</tagNameFormat>
```
Comment (KostyaSha) [inline pom.xml]: Why  with `v`? Why not just use version?

File: pom.xml [hunk 2]
```diff
@@ -282,6 +282,19 @@
...
+                    <mavenExecutorId>forked-path</mavenExecutorId>
+                    <useReleaseProfile>false</useReleaseProfile>
+                    <preparationGoals>clean install</preparationGoals>
```
Comment (KostyaSha) [inline pom.xml]: neve do install when it not required to really install files into maven cache, should be `clean verify`

Comment (lanwen) [inline pom.xml]: copypaste from parent pom

File: pom.xml (hunk 1 above)
Comment (lanwen) [inline pom.xml]: tagname looks better with `v` (imho)
https://github.com/VerbalExpressions/JavaVerbalExpressions/releases

**Pull request merged**